tests/python/frontend/tflite/test_forward.py

```diff
@@ -1850,7 +1850,7 @@ def _test_quantize_dequantize(data):
     # First TFLite quantize op converts float32 tensor to int8 tensor - Qnn quantize.
     # Second TFLite quantize op converts int8 tensor to int8 tensor - Qnn requantize.
     data_in = tf.keras.layers.Input(shape=data.shape[1:])
-    relu = tf.keras.layers.ReLU()(data_in)
+    relu = tf.keras.layers.ReLU()(data)
```

Comment:
Why is this changed? I would suggest to add a new test for quantize. This one takes the Input layer of Keras, previous line.

Reply:
The idea is to use data as constant parameter for quantize operation which will be inserted on quantisation step